homeassistant/components/tellduslive/sensor.py

```diff
@@ -194,4 +194,4 @@ def native_value(self):
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
-        return "{}-{}-{}".format(*self._id)
+        return "-".join(self._id)
```

Comment:
This will throw an error if `self._id` isn't an iterable of strings. The original will not.